playbooks/02 - Use Case - SOC Simulator/Run Scenario - Create Alerts.json

```diff
@@ -4,7 +4,7 @@
     "name": "Run Scenario - Create Alerts",
     "aliasName": null,
     "tag": null,
-    "description": null,
+    "description": "Creates records related to scenario",
```

Comment:
Create*

Reply:
@cs-suyog-jinturkar  Updated PB description to 'Create records related to a scenario'